index.html

```diff
@@ -34,102 +40,107 @@ <h3 class="intro">I'm Bunbee<br>Glad to see<br> you!</h3>
-        <h2 class="card-header">Multi-Post<br>Stories</h2>
-        <ul class="card-links">
-            <li>CANOPY</li>
-            <li> Black End Dev</li>
-            <li>2015</li>
-        </ul>
-        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
-        <div class="lang-section">
-           <ul class="card-lang">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
-           </ul>
-        </div>
-        <button class="btn1">See Project</button>
+        <section class="card-image se">
+            <img class="img-card " src="img/SnapshootPortfolio(4).png" alt="firstoject">
+        </section>
+        <section>
+            <h2 class="card-header">Multi-Post<br>Stories</h2>
+            <ul class="card-links">
+                <li>CANOPY</li>
+                <li> Black End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p class="card-text">A daily selection of privately personalised reads; no accounts or sign-ups required</p>
+            <ul class="card-lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <button class="btn1">See Project</button>
+        </section>
     </section>
-   </div>
-   <div class="about-section">
     <section class="about-me">
-        <h1 class="about-header">About<br> Myself</h1>
-        <p class="about-para"> Hello I'm a software devoloper! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
-        <p class="connect">LET'S CONNECT</p>
-        <ul class="social-media">
-            <li class="fa fa-twitter fa-2x"></li>
-            <li class="fa fa-linkedin fa-2x"></li>
-            <li class="fa fa-medium fa-2x"></li>
-            <li class="fa fa-github fa-2x"></li>
-            <li class="fa fa-facebook fa-2x"></li>
-           </ul>
-        <button class=" btn1 btn2">Get my resume</button>
-
+       <section class="about-first">
+            <h1 class="about-header">About<br> Myself</h1>
+            <p class="about-para"> Hello I'm a software devoloper! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+            <p class="connect">LET'S CONNECT</p>
+            <ul class="social-media">
+                <li class="fa fa-twitter fa-2x"></li>
+                <li class="fa fa-linkedin fa-2x"></li>
+                <li class="fa fa-medium fa-2x"></li>
+                <li class="fa fa-github fa-2x"></li>
+                <li class="fa fa-facebook fa-2x"></li>
+            </ul>
+            <button class=" btn1 btn2 btnp">Get my resume</button>
+       </section>
+       <section class="about-second">
```

Comment:
- [ ] This section is still not meeting all the given design specs.
- Check the length of horizontal lines
- The images of HTML, CSS, JS should not occupy almost all the width of its section.

![Screenshot from 2021-08-28 17-19-46](https://user-images.githubusercontent.com/65501535/131221069-986e11c7-68b9-417c-aa44-5a6f0a677c22.png)

>      ![image](https://user-images.githubusercontent.com/55465133/131048224-afc2a85e-b691-4d40-81cd-d5b0d6d7a7f7.png)